src/components/widgets/sparkline/readme.md

```diff
@@ -0,0 +1,7 @@
+Basic Line Widget:
```

Comment:
they turn in to the documentation at the styleguide site - datavizkit.surge.sh. you can test the theory with `yarn run docs:server`